modules/openfoam/src/OpenFOAM_Registry.txt

```diff
@@ -48,11 +44,11 @@ typedef	OpenFOAM/OpFM	ParameterType	IntKi	NnodesVel	-	-	-	"number of velocity no
 typedef	OpenFOAM/OpFM	ParameterType	IntKi	NnodesForce	-	-	-	"number of force nodes on FAST v8-OpenFOAM interface"	-
```

Comment:
I personally think `nNodesForce` (lower n) would make more sense :)

Reply:
I'm starting to adopt your notation for this ;)

I think I'll hold off on changing this until the next iteration of changes.

Reply:
Changed to `nNodes...`.  A bit more readable this way :)